packages/zowe-explorer/src/dataset/DatasetTree.ts

```diff
@@ -89,7 +89,7 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
             this.mFavoriteSession.iconPath = icon.path;
         }
         this.mSessionNodes = [this.mFavoriteSession];
-        this.treeView = vscode.window.createTreeView("zowe.explorer", { treeDataProvider: this });
+        this.treeView = vscode.window.createTreeView("zowe.explorer", { treeDataProvider: this, canSelectMany: true });
```

Comment:
This is a huge change :( I will have to check every command n the menu, and if it doesn't support it, add an error message.

Not saying it doesn't need to be done. It's just a bit late in the game to be requesting it...

Reply:
Hmm, but with this change, you are basically create a very bad user experience, when the multiselect is working in the tree but may not working in the commands... Maybe we can check for other commands and create an issue or smt like this to track this effort?

Reply:
From what I know, if the context menu does not handle multiple selected nodes, there will be no error messages. It will simply do the action for the actual node you right-clicked on and ignore the other selected nodes.

Reply:
Ok, yes, that's correct. According to the [VSCode docs](https://code.visualstudio.com/api/references/vscode-api#TreeViewOptions%3CT%3E), the first argument to any function will be the tree item the command was executed on. So all the old commands will still receive 1 node, where they are expecting it, and should work as usual.

Reply:
Yep, I understand it. 
The only risk I see here, is that: users may choose a lot of items, issue some command for them and it will affect only one element (the clicked one). I would be very sad :(

But at the same time, multiselection will work for the deletion of the datasets, so I will be sad and confused :(